### PR TITLE
Updates to the DeltaFinder, TZClusterFinder and TEveEventDisplay

### PR DESCRIPTION
**Pull request by NamithaChitrazee**
- Fixed incorrect slope error computation in LsqSums @matthewstortini 

- Fixed a missing parameter initialisation in TZClusterFinder @gianipez 

- Small correction on the CRV 2-D display using TEve @NamithaChitrazee 

- DeltaFinder no longer writes out separate hit flag collections @pavel1murat

### CODE REVIEW

File: CalPatRec/fcl/prolog.fcl [hunk 1]
```diff
@@ -343,8 +343,9 @@ CalPatRec : { @table::CalPatRec
             goodHitMask                 : [ "TimeDivision" ]    ## is there a point in using combohits made
                                                                 ## out of straw hits with an apriory wrong wdist ?
             bkgHitMask                  : [  ]                  ##
-            writeFilteredComboHits      : 0                     ##
-            writeStrawHitFlags          : 1                     ## used by the track fit
+            writeFilteredComboHits      : 0                     ## by default, clone all hits with updated flags
```
Comment (brownd1978) [inline CalPatRec/fcl/prolog.fcl:346]: Are these parameters boolean?  If so, why not initialize with 'true' or 'false'?

File: CalPatRec/src/DeltaFinder_module.cc [hunk 2]
```diff
@@ -58,8 +68,8 @@ namespace mu2e {
       fhicl::Atom<int>             debugLevel             {Name("debugLevel"        )    , Comment("debug level"                ) };
       fhicl::Atom<int>             diagLevel              {Name("diagLevel"         )    , Comment("diag level"                 ) };
       fhicl::Atom<int>             printErrors            {Name("printErrors"       )    , Comment("print errors"               ) };
-      fhicl::Atom<int>             writeFilteredComboHits {Name("writeFilteredComboHits"), Comment("1: write filtered CH coll"  ) };
-      fhicl::Atom<int>             writeStrawHitFlags     {Name("writeStrawHitFlags")    , Comment("1: write SH flag coll"      ) };
+      fhicl::Atom<int>             writeFilteredComboHits {Name("writeFilteredComboHits"), Comment("0: write all CH, 1: write filtered CH") };
```
Comment (brownd1978) [inline CalPatRec/src/DeltaFinder_module.cc:71]: These integers are behaving as booleans, why not define them as such?

File: CalPatRec/fcl/prolog.fcl (hunk 1 above)
Comment (pavel1murat) [reply CalPatRec/fcl/prolog.fcl:346]: see below (:smile:)

File: CalPatRec/src/DeltaFinder_module.cc (hunk 2 above)
Comment (brownd1978) [reply CalPatRec/src/DeltaFinder_module.cc:71]: There is value in declaring variable types explicitly to their purpose.  Here the purpose is to be a boolean, so they should be declared as such.  this has the advantages of creating a standard interface for all Mu2e modules, and allows common-language configuration in the fcl file: ie "true" and "false" vs 1 and 0, which are ambiguous and could mean anything.

Comment (brownd1978) [reply CalPatRec/src/DeltaFinder_module.cc:71]: The issue of correctly typing parameters is outside the scope of this PR.  I will approve and create an issue for this.

**Review by brownd1978 — APPROVED** [review]

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Review by pavel1murat — APPROVED** [review]
the essential part of this PR is getting rid of standalone hit flag collections , the rest is just cosmetics.

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Review by kutschke — APPROVED** [review]
Thanks for fixing the whitespace issues.

**Pull request merged**